lerobot/scripts/push_dataset_to_hub.py

```diff
@@ -182,10 +183,6 @@ def push_dataset_to_hub(
         meta_data_dir = Path(cache_dir) / "meta_data"
         videos_dir = Path(cache_dir) / "videos"
 
-    # Download the raw dataset if available
```

Comment:
Just double checking. Should this really be removed? (I don't actually know)

Reply:
It should have been removed in #231 but I guess we forgot so I took care of it here. If you look a few lines above this condition is now never met because of
```python
if not raw_dir.exists():
    raise NotADirectoryError
```

Reply:
@aliberts dont hesitate to add a comment in the PR so that reviewers dont have to think about this kind of details.